app/components/preprint-form-authors.js

```diff
@@ -131,7 +131,7 @@ export default CpPanelBodyComponent.extend(Analytics, {
                     ));
                     this.highlightSuccessOrFailure(contributor.id, this, 'success');
                 }, (error) => {
-                    if (error.errors[0] && error.errors[0].detail && error.errors[0].detail.indexOf('is already a contributor') > -1) {
+                    if (error.errors[0] && error.errors[0].detail) {
```

Comment:
I lean towards continuing to check for the presence of certain errors in detail, and then instead of just showing the API detail, showing a specific translated phrase to indicate what the error was. 

There is the possibility that the error about adding an unregistered author below will never be caught, if there's an error detail in the response that gets to here. We might end up just showing the API message in all cases.

Reply:
Fair point. It may not always be the case that the API message is understandable to the average user, and just showing the non-translated string goes against intl patterns. Will update